### ISSUE TITLE
Keyboard navigation in WIKI

### ISSUE THREAD

**Issue by PierreDemailly** [NodeSecure/cli#271]
# Feature Request: Keyboard Navigation in Wiki Sections

## Description:
It would be beneficial to have keyboard navigation functionality in the wiki for a smoother user experience. Specifically, the following enhancements are proposed:

### Switch between Flags and SAST Warnings:

Allow users to seamlessly switch between the "Flags" and "SAST Warnings" sections using keyboard shortcuts.

### Flags Section: Move to Previous/Next Flag:

Enable users to navigate efficiently within the "Flags" section by providing keyboard shortcuts to move to the previous and next flag.

### SAST Warnings Section: Move to Previous/Next Warning:

Similarly, introduce keyboard shortcuts to facilitate navigation within the "SAST Warnings" section, allowing users to move to the previous or next warning.

![image](https://github.com/NodeSecure/cli/assets/39910767/4d15cdad-14fd-42d3-af40-12278dbd5837)

**Comment by PierreDemailly** [comment]
Also, could be nice to add a shortcut that open the Wiki.

**Comment by fraxken** [comment]
> Also, could be nice to add a shortcut that open the Wiki.

Done in https://github.com/NodeSecure/cli/issues/271

**Comment by fraxken** [comment]
Should be careful to not mix network keyboard navigation with wiki navigation (probably need to figure some state to lock keydown event in some places).
